tests/migration/migration.go

```diff
@@ -2196,8 +2211,13 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 				dv, err = virtClient.CdiClient().CdiV1beta1().DataVolumes(testsuite.GetTestNamespace(nil)).Create(context.Background(), dv, metav1.CreateOptions{})
 				Expect(err).ToNot(HaveOccurred())
 				libstorage.EventuallyDV(dv, 600, HaveSucceeded())
-				vmi := tests.NewRandomVMIWithDataVolume(dv.Name)
-				tests.AddUserData(vmi, "disk1", "#!/bin/bash\n echo hello\n")
+				vmi := libvmi.New(
+					libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+					libvmi.WithNetwork(v1.DefaultPodNetwork()),
+					libvmi.WithDataVolume("disk0", dv.Name),
+					libvmi.WithResourceMemory("1Gi"),
+					libvmi.WithCloudInitNoCloudEncodedUserData("#!/bin/bash\n echo hello\n"),
```

Comment:
Make this a constant in libvmi factory.go?

https://github.com/kubevirt/kubevirt/blob/9d4a040652074af39d58eeea6a9cf3042b64d285/tests/libvmi/factory.go#L51

Reply:
I added a commit introducing a `BashHelloScript` constant under `cloudinit.go` file

Reply:
But it's not used here?

Reply:
It will also work without it, see comment above.